project/src/main/java/com/google/sps/classes/Block.java

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.classes;
+
+import java.util.List;
+import java.util.ArrayList;
+import com.google.sps.classes.Student;
+import com.google.sps.classes.TimeRange;
+
+public final class Block {
+  
+  private List<Student> students;
+  private final TimeRange time;
+  private final int duration;
+
+  public Block(int start, int duration) {
+
```

Comment:
You have additional whitespace that I don't think you really need? Just makes your file a lot longer than necessary.

Reply:
True I'm gonna trim it, not sure why the formatter did that.